functorch/_src/make_functional.py

```diff
@@ -353,15 +359,22 @@ def compute_loss(params, x, t):
 
     If the model has any buffers, please use :func:`make_functional_with_buffers` instead.
 
+    Args:
+        model (torch.nn.Module): Input model.
+        disable_params_grad (bool): Flag to disable gradients for output parameters.
+            It could be helpful to set this flag to True if gradient computations only use the functorch API.
+            Disabling gradients also helps to avoid accumulating them in the backprop graph and to prevent
+            keeping them in memory. Default: False.
```

Comment:
This is a bit ambiguous. I'd phrase it something like the following:
- The returned params are unrelated to the set of params from the original model. If True (default), the params will have requires_grad=True on them (aka they will be trackable with regular PyTorch autograd), matching the requires_grad-ness of the params from the original model. Otherwise, the returned params will have requires_grad=False.
- If you plan on using regular PyTorch autograd (e.g., if you want to call .backward() or torch.autograd.grad(), then set disable_params_grad=True (default)). Otherwise, if you're only planning on using functorch's gradient transforms, then please set disable_params_grad=False to avoid unnecessarily tracking history with PyTorch autograd.